### DISCUSSION TITLE
Multiple Pollers for a Cluster

### DISCUSSION THREAD

**Discussion by jmg011** [NetApp/harvest#1247] [General]
Hi, 

Would there be any performance impact on a cluster if I run multiple pollers for a same cluster from different servers to collect data with different templates _(dedicated templates per server to avoid duplication)_ ?

**Comment by cgrinds** [comment]
hi @jmg011 - the management plane of the cluster will do more work to serve multiple concurrent pollers, but that effort is usually small. If there are slow downs in the management plane, those typically do not effect the data plane.

**Reply by jmg011** [reply]
Thank you, @cgrinds. 

Assume that I'm setting up two pollers, and both pollers are polling different set of data from a cluster. Will there be an overhead just for serving data to both pollers? If so, do we have any ballpark figure on the overhead utilization?

**Reply by cgrinds** [reply]
Different sets of data help reduce object contention and locking. I've only got anecdotal data from talking with coworkers. I don't have any benchmarks of the impact to share. I'll let you know if I find any.

**Reply by jmg011** [reply]
@cgrinds Sure, thank you.
